### PR TITLE
Update repos overview

### LINKED ISSUE #310: Address the text appearance in the warning alert
Text seems smaller than usual here?

image

### PR DESCRIPTION
**Pull request by emilyvandewalle**
#### Repo overview
image
- Added `queued` column
- Used a 'sum' icon for the `totals` column (an alternative could be the calculator icon from the hero icons as previously mentioned)
- Updated CTA to text for repos without syncs
   - Since we changed the clickable tables to only the link being clickable, I was wondering if a button to set up the sync types makes more sense again in this case?
- Updated clickable table approach for all tables
- Removed empty `Repo settings` action (there currently is no link to go to the repo settings tab directly) and added the delete action into the table as it was the only action and to keep it consistent with the auto import settings table

image

- Added tooltip to github icon
- Also made auto import icon link to the auto import settings. I couldn't figure out how to link it to the auto import detail page which probably makes more sense

image

- Fixed look of sync status dropdown and made it scrollable (added max height)

#### Remove repo dialog
image
- Fixed spacing
- Fixed font size

(Resolves [#221](https://github.com/mergestat/fuse/issues/221))
(Resolves #310)

### CODE REVIEW

File: ui/src/views/repositories/components/repositories-table/index.tsx [hunk 1]
```diff
@@ -1,90 +1,146 @@
...
+            : <div className='flex flex-col min-w-0 bg-white h-full'>
+              <div className='flex-1 overflow-x-auto overflow-y-hidden'>
+                <table data-testid={TEST_IDS.repoTableList} className='t-table-default t-table-clickable'>
+                  <thead>
+                    <tr className='bg-white'>
+                      <th scope='col' key='name' className='whitespace-nowrap'>Repository</th>
+                      <th scope='col' key='last' className='whitespace-nowrap'>Last sync</th>
+                      <th scope='col' key='total' className='whitespace-nowrap w-0 text-center'>Total</th>
+                      <th scope='col' key='queued' className='whitespace-nowrap w-0 text-center'>Queued</th>
+                      <th scope='col' key='loading' className='whitespace-nowrap w-0 text-center'>Loading</th>
```
Comment (patrickdevivo) [inline ui/src/views/repositories/components/repositories-table/index.tsx:47]:
```suggestion
                      <th scope='col' key='running' className='whitespace-nowrap w-0 text-center'>Running</th>
```

**Review by patrickdevivo — CHANGES_REQUESTED** [review]

File: ui/src/pages/settings/auto-imports/auto-import.tsx [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+      type: 'GITHUB_REPO_STARS',
+      description: 'Retrieves all stargazers of a GitHub repo',
+      shortName: 'GitHub Repo Stars',
+    },
+    {
+      type: 'GIT_REFS',
+      description: 'Retrieves all the refs of a git repo',
+      shortName: 'Git Refs',
+    },
+  ]
```
Comment (patrickdevivo) [inline ui/src/pages/settings/auto-imports/auto-import.tsx:79]: Not sure how this is used, but this list should be retrieved dynamically from the server vs hardcoded in here I believe. @german-mergestat

File: ui/src/views/repositories/components/repositories-table/index.tsx [hunk 3]
```diff
@@ -1,90 +1,146 @@
...
+                      <th scope='col' key='error' className='whitespace-nowrap w-0 text-center'>Errors</th>
```
Comment (patrickdevivo) [inline ui/src/views/repositories/components/repositories-table/index.tsx:48]:
```suggestion
                      <th scope='col' key='error' className='whitespace-nowrap w-0 text-center'>Error</th>
```

File: ui/src/views/repositories/components/repositories-table/repositories-table-columns/repository-name.tsx [hunk 4]
```diff
@@ -22,48 +22,55 @@ export const RepositoryName: React.FC<RepositoryNameProps> = (props) => {
...
               placement='bottom'
             >
               {getRelativeTime(props.createdAt)}
             </Tooltip>
           </span>
-          <div className="border-l border-semantic-border px-2">
-            {repoTypeIcon()}
+          <div className='border-l border-semantic-border px-2'>
+            <Tooltip
+              content={`${props.type} repository`}
```
Comment (patrickdevivo) [inline ui/src/views/repositories/components/repositories-table/repositories-table-columns/repository-name.tsx:59]: This appears in the UI as `github repository`:

image

Can we tweak this to handle the capitalization? `GitHub repository`

**Review by patrickdevivo — CHANGES_REQUESTED** [review]

File: ui/src/pages/settings/auto-imports/auto-import.tsx (hunk 2 above)
Comment (german-mergestat) [reply ui/src/pages/settings/auto-imports/auto-import.tsx]: ohh! this is an old file, seems `rebase` brought this through again, but no worries, it is being retrieved from database yep. So, I removed this folder `ui/src/pages/settings/auto-imports`

File: ui/src/views/repositories/components/repositories-table/index.tsx (hunk 3 above)
Comment (german-mergestat) [reply ui/src/views/repositories/components/repositories-table/index.tsx]: Done [eb4649b](https://github.com/mergestat/fuse/pull/391/commits/eb4649bf27c7b3b7f4b96df45b169e9f732f4726)

File: ui/src/views/repositories/components/repositories-table/repositories-table-columns/repository-name.tsx (hunk 4 above)
Comment (german-mergestat) [reply ui/src/views/repositories/components/repositories-table/repositories-table-columns/repository-name.tsx]: Done [eb4649b](https://github.com/mergestat/fuse/pull/391/commits/eb4649bf27c7b3b7f4b96df45b169e9f732f4726)

**Review by patrickdevivo — APPROVED** [review]

**Pull request merged**